index.rst

```diff
@@ -126,7 +126,6 @@ Sensor Components
     INA219, components/sensor/ina219, ina219.jpg
     INA226, components/sensor/ina226, ina226.jpg
     INA3221, components/sensor/ina3221, ina3221.jpg
-    Inkbird IBS-TH1 Mini, components/sensor/inkbird_ibsth1_mini, inkbird_isbth1_mini.jpg
```

Comment:
Please check your commits to not remove unrelated lines

Reply:
update ok